lib/metasploit/framework/data_service/proxy/loot_data_proxy.rb

```diff
@@ -8,19 +8,35 @@ def report_loot(opts)
       end
       data_service.report_loot(opts)
     rescue Exception => e
-      puts "Call to #{data_service.class}#report_loot threw exception: #{e.message}"
+      elog "Call to #{data_service.class}#report_loot threw exception: #{e.message}"
```

Comment:
The message format changed recently in `goliath` branch, update to reflect the new stlye. See other location below.